src/main/java/org/takes/rq/RqMultipart.java

```diff
@@ -569,18 +522,59 @@ private static StringBuilder fakeBody(final Request... parts)
                     this.req,
                     String.format(
                         "Content-Type: multipart/form-data; boundary=%s",
-                        RqMultipart.Fake.BOUNDARY
+                        FakeMultipartRequest.BOUNDARY
                     ),
                     String.format(
                         "Content-Length: %s",
-                        RqMultipart.Fake.fakeBody(this.parts).length()
+                        this.body.length()
                     )
                 ).head();
             }
             @Override
             public InputStream body() throws IOException {
-                return RqMultipart.Fake.fakeStream(this.parts);
+                return new ByteArrayInputStream(
+                    this.body.getBytes(StandardCharsets.UTF_8)
+                );
             }
+
+            /**
+             * Fake body creator.
+             * @param parts Fake request body parts
+             * @return StringBuilder of given dispositions
+             * @throws IOException If fails
+             */
+            @SuppressWarnings("PMD.InsufficientStringBufferDeclaration")
+            private static StringBuilder fakeBody(final Request... parts)
+                throws IOException {
+                final StringBuilder builder = new StringBuilder();
+                for (final Request part : parts) {
+                    builder.append(
+                        String.format("--%s", FakeMultipartRequest.BOUNDARY)
+                        )
+                        .append(RqMultipart.CRLF)
+                        // @checkstyle MultipleStringLiteralsCheck (1 line)
+                        .append("Content-Disposition: ")
+                        .append(
+                            new RqHeaders.Smart(
+                                new RqHeaders.Base(part)
+                            // @checkstyle MultipleStringLiteralsCheck (1 line)
+                            ).single("Content-Disposition")
```

Comment:
@2686747 let's make a constant on class level for `Content-Disposition` header name
and get rid the supression